observability/tools/OpenTelemetry.md

```diff
@@ -0,0 +1,32 @@
+# Open Telemetry
+
+Open Telemetry is a set of APIs, SDKs and instrumentation which can be used for collection, processing and orchestration of telemetry data like traces metrics and logs. It supports multiple languages(Java, dotnet, python, javascript, golang, erlang etc.). Open telemetry follows a vendor agnostic and standards based approach for collection and management of telemetry data. Important point to note is that OpenTelemetry does not have its own backend, all telemetry collected by OpenTelemetry Collector must be sent to a backend like Prometheus etc. Open telemetry is also the 2nd most active CNCF project after Kubernetes.
+
+## Collector
+
+OpenTelemetry Collector performs telemetry collection, processing and export. It is mostly run as an agent or a sidecar with the application or as a daemon if installed directly on host. As the collector is standards based and supports many integrations, there is a possibility that many tool specific agents for metric, traces and logs can be replaced by a single OpenTelemetry collector.
```

Comment:
Inconsistent use of OpenTelemetry in multiple places. We should stick to 'OpenTelemetry'

Reply:
done